resources/compass/charts/kyma-environment-broker/templates/virtualservice.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: {{ include "kyma-env-broker.fullname" . }}
+  labels:
+{{ include "kyma-env-broker.labels" . | indent 4 }}
+spec:
+  gateways:
+  - {{ .Values.global.istio.gateway.name }}
```

Comment:
I think there should be: `'{{ .Values.global.gateway.tls.host }}.{{ .Values.global.ingress.domainName }}'`

Reply:
Or maybe `{{ .Values.global.istio.gateway.name }}.{{ .Values.global.istio.gateway.namespace }}.svc.cluster.local` is fine.